src/components/Wizard/CreateVmWizard/providers/vmware.js

```diff
@@ -0,0 +1,104 @@
+import { get } from 'lodash';
+
+import { isImportProviderType, settingsValue } from '../../../../k8s/selectors';
+import { CONNECT_TO_NEW_INSTANCE, PROVIDER_SELECT_VM } from '../strings';
+import { validateVmwareURL } from '../../../../utils/validations';
+
+import {
+  PROVIDER_VMWARE,
+  PROVIDER_VMWARE_URL_KEY,
+  PROVIDER_VMWARE_USER_NAME_KEY,
+  PROVIDER_VMWARE_USER_PWD_AND_CHECK_KEY,
+  PROVIDER_VMWARE_USER_PWD_REMEMBER_KEY,
+  PROVIDER_VMWARE_VCENTER_KEY,
+  PROVIDER_VMWARE_VM_KEY,
+} from '../constants';
+
+import VMWarePasswordAndCheck from './VMWarePasswordAndCheck';
+import VCenterInstances from './VCenterInstances';
+import { onVCenterInstanceSelected, onVmwareCheckConnection, onVCenterVmSelectedConnected } from './vmwareActions';
+import VCenterVms from './VCenterVms';
+
+const isVmwareNewInstance = basicSettings =>
+  get(basicSettings, [PROVIDER_VMWARE_VCENTER_KEY, 'value']) === CONNECT_TO_NEW_INSTANCE;
+const isNewVmwareInstanceSelected = basicVmSettings =>
+  isImportProviderType(basicVmSettings, PROVIDER_VMWARE) && isVmwareNewInstance(basicVmSettings);
+
+const getVMWareNewConnectionSection = (basicSettings, WithResources, k8sCreate) => ({
+  [PROVIDER_VMWARE_URL_KEY]: {
+    id: 'vcenter-url-dropdown',
+    title: 'vCenter URL',
+    required: true,
+    isVisible: isNewVmwareInstanceSelected,
+    validate: settings => validateVmwareURL(settingsValue(settings, PROVIDER_VMWARE_URL_KEY)),
+    defaultValue: 'https://host:port/',
+    help:
+      'Address to be used for connection to a vCenter instance. The "https://" protocol will be added automatically. Example: "my.domain.com:1234".',
+  },
+  [PROVIDER_VMWARE_USER_NAME_KEY]: {
+    id: 'vcenter-username',
+    title: 'vCenter User Name',
+    required: true,
+    isVisible: isNewVmwareInstanceSelected,
+    help: 'User name to be used for connection to a vCenter instance.',
+  },
+  [PROVIDER_VMWARE_USER_PWD_AND_CHECK_KEY]: {
+    id: 'vcenter-userpwd',
+    title: 'vCenter Password',
+    type: 'custom',
```

Comment:
you can reuse form type constants from `Form/constants`

Reply:
done